modules/nasmediaAdmixerBidAdapter.js

```diff
@@ -58,25 +61,25 @@ export const spec = {
   }
 }
 
-function getOsType() {
-  let ua = navigator.userAgent.toLowerCase();
-  let os = ['android', 'ios', 'mac', 'linux', 'window'];
-  let regexpOs = [/android/i, /iphone|ipad/i, /mac/i, /linux/i, /window/i];
-
-  return find(os, (tos, idx) => {
-    if (ua.match(regexpOs[idx])) {
-      return os[idx];
-    }
-  }) || 'etc';
+function getOs() {
+  let ua = navigator.userAgent;
+  if (ua.match(/(iPhone|iPod|iPad)/)) {
+    return 'ios';
+  } else if (ua.match(/Android/)) {
```

Comment:
Why use `match` instead of `===` for checking if strings are equal? Is this function more efficient?

Same question applies to line 70